A <%= @exception.class %> occurred in <%= @controller_name %>#<%= @controller_action_name %>:

  <%= @exception.message %>
  <%= @backtrace.first %>

<%= @sections.map { |section| render_section(section) }.join %>
